### PR TITLE
Update django-webpack-loader from 1.3.0 to 1.4.0

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

This update seems to introduce some problems, so we have to figure out how to fix the tests.
I assume it has something to do with https://github.com/django-webpack/django-webpack-loader/pull/297.

### Proposed changes

- Update django-webpack-loader from 1.3.0 to 1.4.0

### CODE REVIEW

**Comment by melegiul** [comment]
The new webpack loader version assumes that the key "request" is set in the templates context. As far as I know this happens per default in most circumstances, but in case of our pdf export we creating a custom context which does not include the request key.
So one way to fix this would be to add a request object to the pdf context, which is passed by the view over here: https://github.com/Integreat/integreat-cms/blob/3632dd978cac673809cb72077aee1dc80c4c774b/src/cms/utils/pdf_utils.py#L85
If you agree, I could also push a commit for this?

**Comment by timoludwig** [comment]
> The new webpack loader version assumes that the key "request" is set in the templates context. As far as I know this happens per default in most circumstances, but in case of our pdf export we creating a custom context which does not include the request key.
> So one way to fix this would be to add a request object to the pdf context, which is passed by the view over here:

Ah okay. Do you think it's sufficient to add a dummy request object with `context["request"] = object()`? Then we don't have to refactor as much...
 
> If you agree, I could also push a commit for this?

Sure, go ahead!

**Comment by timoludwig** [comment]
This also affects out error templates which are rendered via [render_to_string()](https://docs.djangoproject.com/en/3.2/topics/templates/#django.template.loader.render_to_string) without the request context.

**Comment by melegiul** [comment]
Yes, dummy request is possible too, but object() does not allow setting attributes, which is required, so I used type() instead

File: src/cms/views/error_handler/error_handler.py [hunk 1]
```diff
@@ -22,6 +22,15 @@ def render_error_template(context):
...
     """
+    # django-webpack-loader assumes request object in context
+    # and that this request object has a specific attribute set
+    context.update(
+        {
+            "request": type(
+                "dummy_request", (object,), {"_webpack_loader_used_tags": set()}
+            )()
+        }
+    )
```
Comment (timoludwig) [inline src/cms/views/error_handler/error_handler.py:33]: I think in this case we should just use the real request of the `handler400()`, `handler403()` etc...

File: src/cms/utils/pdf_utils.py [hunk 2]
```diff
@@ -82,13 +82,19 @@ def generate_pdf(region, language_slug, pages):
             title = region.name
     language = Language.objects.get(slug=language_slug)
     filename = f"Integreat - {language.translated_name} - {title}.pdf"
+    # django-webpack-loader assumes request object in context
+    # and that this request object has a specific attribute set
+    dummy_request = type(
+        "dummy_request", (object,), {"_webpack_loader_used_tags": set()}
+    )()
```
Comment (timoludwig) [inline src/cms/utils/pdf_utils.py:89]: `dummy_request = type("", (), {})()` should be enough, right? The attribute `_webpack_loader_used_tags` is set by webpack loader if not existent on the request object anyway...
Or, maybe even better: do an `from django.http import HttpRequest` and use `HttpRequest()` as dummy request? maybe a bit more readable...

Comment (melegiul) [reply src/cms/utils/pdf_utils.py]: Ok, I took the real request for error templates and a dummy HttpRequest object for the PDF export

**Comment by timoludwig** [comment]
@melegiul Thanks a lot! I just resolved the conflicts and added the request object to the context directly instead of passing it to the function and then updating the context.
Can you have a look at it again and approve it if you're ok with the changes?

**Review by melegiul — APPROVED** [review]
Thank you, looks great now :+1:

**Pull request merged**